### ISSUE TITLE
안녕하세요! 질문 있어서 남깁니다.

### ISSUE THREAD

**Issue by ProtossDragoon** [JunHyeok96/Road-Segmentation#1]
https://sol2gram.tistory.com/12?category=816331 블로그 보고 알게 되었습니다.

다름 아니라, 모델을 사용해 보다가 궁금한 점이 생겨서 이슈로 남깁니다.

직접 만드신 구조인 PSP-UNet 을 transfer-learning 으로 학습시킨 것인가요?

처음에 가중치초기화를 어떻게 하고 학습시키신 것인지 (어떤 pretrained weight 를 사용하신 것인지) 궁금합니다!

감사합니다.

**Comment by JunHyeok96** [comment]
안녕하세요~
pspunet은 vgg16으로 transfer-learning한 구조와 [pspuet-vgg16-transfer-learning](https://github.com/JunHyeok96/Road-Segmentation/blob/master/model/pspunet_vgg16.py) transfer-learning을 적용하지 않은 구조[pspuet-vgg16](https://github.com/JunHyeok96/Road-Segmentation/blob/master/model/pspunet.py) 둘 다 테스트해봤을때 transfer-learning을 한 모델의 성능이 더 낮게 나왔습니다. 그렇기에 readme에 작성된 모델은 transfer-learning을 적용하지 않은 모델입니다. 조심스럽게 예상해보기로는 transfer learning에 사용된 vgg16의 데이터셋 도메인과 해당 데이터 셋의 차이가 발생한게 아닌가 싶습니다. 이부분은 정확히 말씀드리기 힘드네요. 가중치 초기화의 경우 사용하지 않았는데, 그 이유는 batch-normalization을 적용했기 때문입니다. bacth-norm의 논문을 읽어보면 가중치 초기화가 큰 의미가 없다고 나와있습니다. 감사합니다.

**Comment by ProtossDragoon** [comment]
앗! 빠르게 답변해 주셔서 정말 감사합니다.

vgg16 의 pretrained 과 풀려고 하는 문제의 도메인 차이 때문일 것이라고 말씀 주셨는데, 실험하셨을 때 사용하셨던 vgg16 의 pretrained 가 imagenet 으로 학습된 weight 인가요?

<hr>

질문을 하는 이유는, 저도 모델을 학습시켜서 사용을 해 보려고 하던 참에 kitty pretrained 를 사용해서 AIhub 데이터셋을 학습시키면 조금 더 나은 성능을 낼 수 있지 않을까 하는 생각이 들어서입니다. 혹시 실험에 사용하셨던 `vgg_model.load_weights("./model/vgg16.h5")` transfer-learning 모델의 가중치가 kitty pretrained 이라면 굳이 준혁님이 선행하셨던 내용을 답습할 필요가 없지 않을까... 하는 생각이 들었습니다.

**Comment by JunHyeok96** [comment]
imagenet기반입니다!
